MTGToolbox.Web/MTGToolbox.Web.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="Microsoft.AspNetCore.All" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="2.2.2" />
```

Comment:
This is the nuget package that enables lazy loading. As the name implies, it's using proxies to accomplish this.